SharePointFramework/PortfolioWebParts/src/components/LatestProjects/LatestProjects.module.scss

```diff
@@ -1,22 +1,35 @@
+a {
```

Comment:
Kommer ikke dette automatisk med <Link > fra Fluent UI React components?

Reply:
Ser ikke sånn ut... Tror det har noe med tema å gjøre...

Reply:
Synes vi bør ha en klasse for lenkene istedenfor generell styling for a, men det er minor. Skal se videre på PRen senere.

Reply:
Enig, er det noe vi kunne hatt i shared? delt styling, lenker vil være et tilfelle ihvertfall

Reply:
Link fra fluent ui react components skal jo følge tema, så veldig synd om vi trenger å overskrive. Hvordan ser de ut uten stylingen din?

Reply:
Ingen forskjell! Tror det er noe jeg gjorde med gamle Link fra gamle fluent ui og trodde jeg trengte det når jeg gikk over til react components, fikser kjapt.